CHANGE_LOG

```diff
@@ -1,3 +1,33 @@
+Version 0.55.1 (27 January, 2022)
+---------------------------------
+
+This is a bugfix release that closes all the remaining issues from the
+accelerated release of 0.55.0 and also any release critical regressions
+discovered since then.
```

Comment:
```suggestion
discovered since then.

CUDA target deprecation notices:

* Support for CUDA toolkits < 10.2 is deprecated and will be removed in Numba
  0.56.
* Support for devices with Compute Capability < 5.3 is deprecated and will be
  removed in Numba 0.56.

```